experimental/await-ops.md

```diff
@@ -0,0 +1,22 @@
+# [Await.ops](https://github.com/tc39/proposal-await.ops)
+
+## AwaitExpression
+
+```js
+interface AwaitExpression <: Expression {
+    type: "AwaitExpression";
+    argument: Expression;
+    operation: Identifier | null
+}
+```
```

Comment:
As now the operation is an `Identifier`, can you add a comment that the `operation.name` must be one of `"all" | "allSettled" | "race" | "any"`?

Reply:
I added an inline comment , let me know if that's okay.